goltsmannn2/part3/client.cpp

```diff
@@ -0,0 +1,35 @@
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <stdio.h>
+#include <protobuf.pb.h>
+
+#define PORT 8000
```

Comment:
typically it is best to not use `#define` for variables, you can use `const` or `static const` for these, or for this you can just set the number directly without making a `#define`.